### ISSUE TITLE
pip install httpony is broken

### ISSUE THREAD

**Issue by patrickpierson** [mblayman/httpony#10]
If i run 

> pip install httpony

 it downloads 

```
Pygments-2.0.2
Werkzeug-0.10.4
httpie-0.9.2
requests-2.7.0
```

which causes the following when hitting the httpony server.

```
Internal Server Error

The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.
```

**Comment by patrickpierson** [comment]
More information.  Downloading the git repo and installing the requirements.txt frozen versions of those repos works fine.

**Comment by mblayman** [comment]
Looks like Werkzeug changed its API.

```
 * Running on http://localhost:8000/ (Press CTRL+C to quit)
127.0.0.1 - - [20/Oct/2016 10:35:09] "GET / HTTP/1.1" 500 -
Error on request:
Traceback (most recent call last):
  File "/Users/matt/.virtualenvs/httpony/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/Users/matt/.virtualenvs/httpony/lib/python2.7/site-packages/werkzeug/serving.py", line 181, in execute
    application_iter = app(environ, start_response)
  File "/Users/matt/projects/httpony/httpony/application.py", line 38, in application
    stream = streams.build_output_stream(args, env, prepared, response=None)
TypeError: build_output_stream() takes exactly 5 arguments (4 given)
```

**Comment by mblayman** [comment]
Correction: it's httpie that changed.

**Comment by mblayman** [comment]
I released version 0.2 which includes this fix. Thanks, @patrickpierson!
